dfttopif/drivers.py

```diff
@@ -109,18 +110,19 @@ def directory_to_pif(directory, verbose=0, quality_report=True, inline=True):
     '''
 
     # Look for the first parser compatible with the directory
-    foundParser = False
-    for possible_parser in [VaspParser, PwscfParser]:
+    found_parser = False
+    for possible_parser in [PwscfParser, VaspParser]:
         try:
             parser = possible_parser(directory)
-            if parser.test_if_from(directory):
-                foundParser = True
-                break
-        except: pass
-    if not foundParser:
+            found_parser = True
+            break
+        except InvalidIngesterException:
+            # Constructors fail when they cannot find appropriate files
+            pass
+    if not found_parser:
         raise Exception('Directory is not in correct format for an existing parser')
     if verbose > 0:
-        print("Found a %s directory", parser.get_name())
+        print("Found a %s directory" % parser.get_name())
```

Comment:
fwiw, I'd prefer to standardize on the new `.format()` string formatting syntax, but :man_shrugging: .

Reply:
👍 I'll make the change

Reply:
:100:

Reply:
Yadu sold me on the advantages of `.format()`. I just keep forgetting to use it.

Reply:
Thanks @yadudoc !